pupil-spa/src/app/check/check.component.ts

```diff
@@ -203,6 +227,10 @@ export class CheckComponent implements OnInit {
         this.auditService.addEntry(new CheckSubmissionPending());
         this.isWarmUp = false;
         this.viewState = 'submission-pending';
+        this.window.ga('send', {
+          hitType: 'pageview',
+          page: '/submission-pending'
```

Comment:
Possibly these URLs should all be under the '/check/' prefix.  Would that make more sense to GA?

Reply:
@jon-shipley If possible I'd like to avoid the check prefix and use something else.
Regexps don't work as I was expecting on excludedRoutes and since I have excluded the keyword 'check' that won't work well.